### PR TITLE
Use the first output for each input, not the last

### PR DESCRIPTION
**Pull request by dobesv**
When an input resulted in multiple outputs (e.g. a `.js` and a `.map` file) the old algorithm kept the latter output `.map`.  However, for `js` files at least, the first output is more interesting since it's the javascript file.

Fixes https://github.com/mugi-uno/parcel-plugin-bundle-manifest/issues/8

### CODE REVIEW

**Comment by mugi-uno** [comment]
I confirmed this problem.
I will merge if the conflict is resolved.

File: plugins/BundleManifestPlugin.js [hunk 1]
```diff
@@ -47,12 +49,14 @@ module.exports = function (bundler) {
...
+    const manifestValue = {}
 
     logger.status('📦', 'PackageManifestPlugin');
     feedManifestValue(bundle, manifestValue, publicURL);
     logger.status('📄', `manifest : ${manifestPath}`);
 
-    fs.writeFileSync(manifestPath, JSON.stringify(manifestValue));
+    const oldManifestValue = readManifestJson(manifestPath);
+    const combinedManifest = Object.assign(oldManifestValue, manifestValue)
+    fs.writeFileSync(manifestPath, combinedManifest);
```
Comment (mugi-uno) [inline plugins/BundleManifestPlugin.js]: oh, please use `JSON.stringify`

Comment (dobesv) [reply plugins/BundleManifestPlugin.js]: Whoooopss!

**Comment by dobesv** [comment]
OK I fixed that issue, can you try again ?

**Comment by mugi-uno** [comment]
I confirmed that the problem will be fixed by this PR.
thanks!

**Pull request merged**